src/gg.regression.unity.bots/Runtime/Scripts/RGBots.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using RegressionGames;
+using UnityEngine;
+
+public static class RGBots
+{
+    private static IRGBotEntry FindBot(string botName)
+    {
+        var botList = Resources.Load<IRGBotList>("RGBotList");
+
+        var botEntry = botList.botEntries.FirstOrDefault(b => b.botName == botName);
+        if (botEntry != null)
+        {
+            return botEntry;
+        }
+        return null;
+    }
+
+    private static IRGBotEntry FindBotByType<T>() where T : Component
+    {
+        var assemblyQualifiedName = typeof(T).AssemblyQualifiedName;
+        var botList = Resources.Load<IRGBotList>("RGBotList");
+
+        var botEntry = botList.botEntries.FirstOrDefault(b => b.qualifiedName == assemblyQualifiedName);
+        if (botEntry != null)
+        {
+            return botEntry;
+        }
+        return null;
+    }
+
+    public static GameObject SpawnBot<T>() where T : Component
+    {
+        GameObject botGameObject = new GameObject(typeof(T).Name);
+        var botEntry = FindBotByType<T>();
+        var behaviorName = botEntry?.botName ?? "Empty";
+        if (botEntry != null)
+        {
+            botGameObject.AddComponent<T>();
+        }
+        LinkToOverlay(botGameObject, behaviorName);
+        return botGameObject;
+    }
+
+    public static GameObject SpawnBot<T>(GameObject prefab) where T : Component
```

Comment:
Just realized, for these, should we also be providing a position vector? 

Also, before we get to merging, these functions specifically should have comments since they will be user-facing.